frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -47,6 +49,8 @@ local symbol_prefix = {
         pages_left = "=>",
         -- @translators This is the footer letter prefix for battery % remaining.
         battery = C_("FooterLetterPrefix", "B:"),
+        -- @translators This is the footer letter prefix for bookmarks count.
```

Comment:
That should be "bookmark count" but I think "number of bookmarks" would be clearer.
```suggestion
        -- @translators This is the footer letter prefix for the number of bookmarks (bookmark count).
```